### PR TITLE
XIVY-9123 Add NN for multi app

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

File: src/app/pages/news/9.4/02-multi-app-context.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-- __Project Tree__: All configuration files in the Axon Ivy project are located under the `Config` folder both in Axon Ivy Designer and physically on the file system - full transparency.
+- __Feature Driven Development__: It is no longer necessary to pack everything into one application and have a risk of clumping. Different sub-applications can be developed in independent applications and still have the same user and role base.
+
+- __Independent release cycles__: By splitting your application into multiple applications, you can develop each application independently and maintain an independent release cycle.
+
+- __Standalone Portal__: The portal no longer needs to be part of your application. Run the portal in its own application and integrate your business processes using the iFrame approach and keep the portal up-2-date and leave all migration pain 
+behind.
+
+- __Multi Tenancy__: For multi-tenancy, we strongly recommend to run a separate Axon Ivy Engine per tenant and to orchestrate this in a container platform. If you want to run multi-tenancy on one engine, then we recommend the path of one or more applications per tenant with independent security systems.
```
Comment (phochstr) [inline src/app/pages/news/9.4/02-multi-app-context.md:12]: This is not clear to me: 

Do you suggest to
- use one security system per tenant for all of the tenant's applications
- have a set of applications with their independent security systems per tenant?

I'd write:
... then we recommend one security system per tenant that is used by all of the applications of the respective tenant.

**Review by phochstr — APPROVED** [review]
Decide on the wording in __Multi Tenancy__, then merge.

File: src/app/pages/news/9.4/02-multi-app-context.md (hunk 1 above)
Comment (alexsuter) [reply src/app/pages/news/9.4/02-multi-app-context.md:12]: Intention: one security system per tenant. but you can have more than one app per tenant.

**Comment by alexsuter** [comment]
@phochstr is it better now?

**Pull request merged**